setup/src/shell

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# echo
```

Comment:
This could probably be removed since it's in a file now.

Reply:
Let's do a second PR for this :)